apps/checker/app/services/EntityHelper.scala

```diff
@@ -34,8 +38,11 @@ class EntityHelper(wsClient: WSClient, config: CheckerConfig) {
       )
       .post(body)
       .map { response =>
+        val after = System.currentTimeMillis
         response.status match {
           case 200 => {
+            logger.info(s"Request to ${config.nerApiUrl} succeeded in ${after - before}ms")
```

Comment:
Might be worth adding markers to this log line so we can measure latency as a separate field in ELK – [here's an example.](https://github.com/guardian/typerighter/blob/2181de7552016cfa3259dbccc58dc69d8f1a269c/apps/checker/app/services/MatcherPool.scala#L51)

Reply:
Ah good shout. The `Timer` class which seems to do exactly what we need here. See: https://github.com/guardian/typerighter/pull/453/commits/08dec4c5034e92ede157a9581fbeafd984292620?w=1 (best viewed with whitespace hidden)